### ISSUE TITLE
plan to make a gui

### ISSUE THREAD

**Issue by jjhesk** [sec-bit/zkPoD-node#13]
are their any plans to build a frontend to visualize this idea?

**Comment by p0n1** [comment]
@jjhesk Yes. We have made some simple design but with a lower priority in our plan by now. Do you have any suggestions?

**Comment by jjhesk** [comment]
a simple k-line, depth view, and buy sell buttons
